theme/themes/pxt/collections/menu.overrides

```diff
@@ -76,4 +76,15 @@
 
 .debugger .ui.inverted.menu {
     background: @DebuggerBackgroundPrimaryColor !important;
+}
+
+/* Debug mode */
+.menubar .debugger-menu-item.centered {
+    position: fixed;
+    place-content: center;
+    right: 0;
+    left: 0;
+    margin-right: auto;
+    margin-left: auto;
+    min-height: inherit;
```

Comment:
Why is this not in `debugger.less`

Reply:
I thought the top bar css was here. I can move it if you think it should go there.